pepper-apis/checkstyle.xml

```diff
@@ -222,7 +222,7 @@
             <property name="target" value="CLASS_DEF, INTERFACE_DEF, ENUM_DEF, METHOD_DEF, CTOR_DEF, VARIABLE_DEF"/>
         </module>
         <module name="JavadocMethod">
-            <property name="scope" value="public"/>
+            <property name="accessModifiers" value="public"/>
```

Comment:
Are these the same thing but with different names?

Reply:
Yes here is a release note from a version just after the one we were using
![Screenshot 2024-04-11 at 2 39 50 PM](https://github.com/broadinstitute/ddp-study-server/assets/47120870/24ef5992-dcde-469a-b3d6-9ac343d33d3d)
And also here is a link to the doc
[https://checkstyle.sourceforge.io/checks/javadoc/javadocmethod.html](https://checkstyle.sourceforge.io/checks/javadoc/javadocmethod.html)